### PR TITLE
fix the old interface causing a build error

### PR DESCRIPTION
**Pull request by bradygaster**

### CODE REVIEW

File: StreamR.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
```
Comment (davidfowl) [inline StreamR.csproj:5]: This entire attribute should be removed.

File: Startup.cs [hunk 2]
```diff
@@ -34,7 +33,7 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment environment)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment environment)
```
Comment (davidfowl) [inline Startup.cs:36]: The old code is right. This should be reverted.

Comment (BrennanConroy) [reply Startup.cs:36]: @bradygaster You probably need to use a newer aspnet core runtime

Comment (bradygaster) [reply Startup.cs:36]: Sorry about that - noticed a build error and went the wrong way on it. I'll update my own environment this time. :)

File: StreamR.csproj (hunk 1 above)
Comment (bradygaster) [reply StreamR.csproj:5]: Yes this was added so I could publish to App Service using AzDO to show preview builds prior to ANCM being updated.

**Pull request closed without merging**